package.json

```diff
@@ -1,6 +1,11 @@
 {
+  "scripts": {
+    "start": "nodemon bot.js"
```

Comment:
is bot.js correct? what is this referring to?

Reply:
It should be index.js

Reply:
alright change that and it will be good

Reply:
@moussaTayeb